configs/defs/generic_linux.textproto

```diff
@@ -576,6 +582,7 @@ benchmark_configs: {
   compliance_note: {
     version: { cpe_uri: "cpe:/o:cos:cos_linux:89" version: "1.0.0" benchmark_document: "CIS Container-Optimized OS" }
     version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
```

Comment:
Let's keep the ordering consistent - either add this always below centos or always above cos.

While we're at it might want to sort the versions alphabetically by CPE URI

Reply:
Is there an automated way to sort all the versions?

Reply:
Well in emacs one could use keyboard macros, not sure how other people do these things :)

But it's also enough if you can just put them to a consistent location, I might do a proper sort in a followup commit.

Reply:
Everything should be sorted now